aptos-move/block-executor/src/view.rs

```diff
@@ -219,6 +243,58 @@ impl<'a, T: Transaction, S: TStateView<Key = T::Key>, X: Executable> LatestView<
         }
         ret
     }
+
+    #[allow(unused)]
+    /// Same as `get_base_value`, but also tries to lift values.
+    fn get_base_value_with_layout(
```

Comment:
it is hard to review without actual place this will be called from being connected

it seems very confusing to me that calling "with_layout" function variant unconditionally performs the exchange. 

exchange will be gated by the AGGREGATOR_EXECUTION flag, and so if the caller will do:

```
if flags.aggregator_execution_enabled() {
   view. get_base_value_with_layout() 
} else {
   view. get_base_value() 
 }
```

I would rename this function to get_base_value_with_aggregator_v2_exchange()


if on the other hand, flag gating check will be performed inside this function - and get_base_value_with_layout will always be called, then function name is ok.